### ISSUE TITLE
Dropped Frames

### ISSUE THREAD

**Issue by agouaillard-cosmo** [CoSMoSoftware/OBS-studio-webrtc#232]
It looks like like the dropped frame counters is cumulative.
we should make sure it is reset from one call to the next.

**Comment by murillo128** [comment]
also dropped frames names is misleading as it calculates the FIR/PLI requests when on webrtc, not sure if we can change the label.

**Comment by agouaillard-cosmo** [comment]
that s requests from the server side right? not dropped frames from OBS.

**Comment by ludocosmo** [comment]
Tests on OBS m84-v23.2 Linux
- Start OBS
- Start streaming to Millicast in VP8 and wait for dropped frames counter to be > zero
- Stop streaming
- Start streaming to Millicast again in VP8, the dropped frame counter is back to zero
- Stop OBS
==============================
- Start OBS
- Start streaming to Millicast in VP9 and wait for dropped frames counter to be > zero
- Stop streaming
- Change codec to VP8
- Start streaming to Millicast in VP8, the dropped frame counter is back to zero
- Stop OBS
==============================
end of tests

**Comment by SCG82** [comment]
> also dropped frames names is misleading as it calculates the FIR/PLI requests when on webrtc, not sure if we can change the label.

I believe that label is set here: https://github.com/CoSMoSoftware/OBS-studio-webrtc/blob/m84-v23.2-simulcast/UI/data/locale/en-US.ini#L35 but can also be directly set here: https://github.com/CoSMoSoftware/OBS-studio-webrtc/blob/m84-v23.2-simulcast/UI/window-basic-stats.cpp#L107  if localization isn't desired.

**Comment by SCG82** [comment]
> It looks like like the dropped frame counters is cumulative.
> we should make sure it is reset from one call to the next.

In my fork I call
```
void WebRTCStream::ResetStats()
{
    started_            = true;
    stopping_           = false;
    frame_id_           = 0;
    video_bytes_sent_   = 0;
    audio_bytes_sent_   = 0;
    video_bytes_rtx_    = 0;
    media_st_timestamp_ = 0;
    outbound_timestamp_ = 0;
    frames_dropped_     = 0;
    frames_sent_        = 0;
    huge_frames_sent_   = 0;
    packets_lost_       = 0;
    packets_sent_       = 0;
    nack_received_      = 0;
    pli_received_       = 0;
    prev_video_bytes_   = 0;
    prev_frames_sent_   = 0;
}
```
near the beginning of `WebRTCStream::Start()`

**Comment by murillo128** [comment]
we should remove the stat, it is misleading, causing confusion and provides no useful information to the user.

**Comment by agouaillard-cosmo** [comment]
merged into #258
